parsl/launchers/launchers.py

```diff
@@ -6,9 +6,16 @@
 logger = logging.getLogger(__name__)
 
 
+def bool_to_num(b: bool):
```

Comment:
can you do this instead?
```
>>> print(int(True))
1
>>> print(int(False))
0
```

Reply:
If official python documentation says that this will always be the case, yes. On the other hand, the existing code does not require consulting anything else in order to ensure that (False, True) -> (0, 1).

Can we stop wasting time on such things? I am of the opinion that, as part of the review process, providing equivalent code that maybe to somebody looks better without justifying why it is so, is a pretty counterproductive way of going about it.

Reply:
Docs do say that -> https://docs.python.org/release/3.0.1/reference/datamodel.html#the-standard-type-hierarchy. We just need to be careful not to do str(Bool) which will give you "True"/"False". bool_to_num is redundant.

Reply:
Did both/either of you *knew* (as opposed to suspected) this before this conversation?

Reply:
Yes. I think the code will be cleaner without this function.

Reply:
Can you *please* stop commenting on matters of style and making things slightly cleaner. I am asking because this has been happening consistently, it annoys the crap out of me, like really badly, while at the same time we miss actual serious bugs in the code.